mtp_api/apps/security/management/commands/bulk_unmonitor.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand, CommandError
+
+from security.models import SavedSearch, PrisonerProfile, BankAccount, DebitCardSenderDetails
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+        parser.add_argument('user', help='User id or username')
+        # TODO: add an option to move monitoring to a different user? requires deduplication
+
+    def handle(self, user, **options):
+        user = find_user(user)
+        if not user:
+            raise CommandError(f'User "{user}" not found')
+        self.stdout.write(f'Removing items monitored by {user.username} ({user.email})')
+
+        self.delete(SavedSearch, 'saved searches', user)
+        self.delete(PrisonerProfile.monitoring_users.through, 'monitored prisoners', user)
+        self.delete(BankAccount.monitoring_users.through, 'monitored bank account senders', user)
+        self.delete(DebitCardSenderDetails.monitoring_users.through, 'monitored debit card senders', user)
```

Comment:
Just to be sure this is doing what I think it's doing: This is deleting the many-to-many records associated with the user, e.g. `BankAccount` <-> `User`, not the `BankAccount` itself right? (I think it's the case but double checking)

Reply:
bingo! django calls it a "through" model which, in all these cases, are automatically generated